src/Illuminate/Auth/Passwords/TokenRepositoryInterface.php

```diff
@@ -23,6 +23,14 @@ public function create(CanResetPasswordContract $user);
      */
     public function exists(CanResetPasswordContract $user, $token);
 
+    /**
+     * Determine if the given user recently created a password reset token.
+     *
+     * @param  \Illuminate\Contracts\Auth\CanResetPassword  $user
+     * @return bool
+     */
+    public function recentlyCreatedToken(CanResetPasswordContract $user);
```

Comment:
would it not be a breaking change for those implementing this interface?

Reply:
If it's going to master and master represents the next major version, it should be ok.